views/ipv/page/photo-id-security-questions-find-another-way.njk

```diff
@@ -0,0 +1,68 @@
+{% extends "shared/base.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% set pageTitleKey = 'pages.photoIdSecurityQuestionsFindAnotherWay.title' | translateWithContext(context) %}
+{% set googleTagManagerPageId = "photoIdSecurityQuestionsFindAnotherWay" %}
```

Comment:
Turns out we don't need googleTagManagerPageId anymore
https://gds.slack.com/archives/C05811D4JMR/p1736762317208379?thread_ts=1736523057.017409&cid=C05811D4JMR